modules/ROOT/pages/design-storage-dc.adoc

```diff
@@ -0,0 +1,40 @@
+= Storing Projects in Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: design-branching.adoc
+
+:imagesdir: ../assets/images
+
+By default, the projects that you work on in API Designer are saved in Design Center. Only other users in your Anypoint Platform business organization can edit such projects.
+
+
+When you store in Design Center a project that you work on in API Designer's text editor (either an API-specification project or an API-fragment project), the project is saved to the branch `master` by default. You can create branches of the project if you do not want to change the content that is in the `master` branch. You can use all of the text editor's features when you are working in a branch. You can even publish the content of the branch as an asset in Anypoint Exchange.
+
+[IMPORTANT]
+====
+You cannot merge changes made in one branch into another branch. For example, if you create the branch `prod-api-1` from your default branch, you cannot merge your changes in `prod-api-1` into the default branch. If you create the branch `prod-api-2` from `prod-api-1`, you cannot merge your changes in `prod-api-2` into `prod-api-1`. Branches are always independent of each other.
+====
+
+When you create a branch, the text editor copies into the new branch the current content of the branch that you are in. For example, if you are in the `master` branch and you create the branch `branch-1`, the current content of `master` is copied into `branch-1`. If you subsequently create the branch `branch-2`, the current content of `branch-1` is copied into `branch-2`.
```

Comment:
Is this explanation necessary? Won't the audience understand what "creating a branch" entails? I do, and I am not as technical as they.